src/Files.Uwp/UserControls/AddressToolbar.xaml

```diff
@@ -411,7 +411,7 @@
                     Height="32"
                     x:FieldModifier="public"
                     AccessKey="R"
-                    AutomationProperties.Name="{helpers:ResourceString Name=NavRefreshButton/AutomationProperties/Name}"
+                    AutomationProperties.Name="{helpers:ResourceString Name=NavResfreshButton/AutomationProperties/Name}"
```

Comment:
This resource string is defined as NavRe*s*freshButton. A separate PR should change all tranlations files to NavRefreshButton.